docs/sp-add-ins/sharepoint-admin-apis-authentication-and-authorization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SharePoint admin APIs authentication and authorization
+description: This article describes the authentication and authorization model for SharePoint admin APIs.
+author: samkabue
+ms.author: samkabue
+ms.date: 06/28/2024
+ms.localizationpriority: high
+ms.service: sharepoint
+---
+
+# SharePoint admin APIs authentication and authorization
+
+SharePoint admin APIs are accessible via [SharePoint client object model (CSOM)](./complete-basic-operations-using-sharepoint-client-library-code.md), [SharePoint REST service](./get-to-know-the-sharepoint-rest-service.md), and [Microsoft Graph](../apis/sharepoint-rest-graph.md).
+
+## Overview
+
+Here are some key principles about SharePoint admin APIs authentication and authorization:
+
+- Admin APIs can be called by administrators (including users with [granular delegated admin privileges (GDAP)](/partner-center/gdap-supported-workloads#sharepoint)) and applications registered in Microsoft Entra.
+- Each admin API defines the least-permissive permission a caller should have to call the API.
+- We use role-based access control to assert a caller can call a particular admin API. If the caller has not been granted the required permission, they'll get an unauthorized access error.
+
+## Pre-Requisites
+
+- Caller is a user with an assigned administrator role in Microsoft Entra, or
+- Caller is a registered Microsoft Entra application with the right application permissions.
+
+## Authorization
+
+SharePoint admin APIs can be called by users with assigned administrator roles in Microsoft Entra, or by applications registered in Microsoft Entra configured with [access on behalf of a user](/graph/auth-v2-user) or [access without a user](/graph/auth-v2-service).
+
+### Access via administrator role
+
+Users in a tenant can be granted administrator roles via [Microsoft admin center](/microsoft-365/admin/add-users/about-admin-roles) or [Microsoft Entra](/entra/fundamentals/users-assign-role-azure-portal). Not all administrator roles have access to all the admin APIs, and some administrator roles do not have access to any admin APIs at all. For example, users with [Global Reader](/entra/identity/role-based-access-control/permissions-reference#global-reader) role have access to read admin APIs, but have no access to admin APIs that make any updates.
+
+### Application access on behalf of a user
+
+Admin API operations [on behalf of a user](/graph/auth-v2-user) require applications to receive consent for SharePoint `AllSites.FullControl` application permission. This permission requires admin consent on the consuming tenant before any user from the tenant can consent to it.
+
+### Application access without a user
+
+Admin API operations [without a user](/graph/auth-v2-service) require applications to receive consent for Microsoft Graph [`Sites.FullControl.All`](/graph/permissions-reference#sitesfullcontrolall) application permission. This permission requires admin consent on the consuming tenant.
```

Comment:
The link to the Sites.FullControl.All role is wrong. It points to the Graph role. But the accepted role is actually on the SharePoint resource and not on Graph. If they request the Graph role, the call will fail due to invalid audience. 

That said, the link this points to has a nice way of documenting the scopes. You should consider adding that for SharePoint’s AllSites.FullControl and Sites.FullControl.All

Reply:
@dluces let me know if the updated text makes sense now. I like the idea of a page documenting the scopes -- let me see if I can do that in this same PR.

Reply:
For the scopes documentation I've added this as part of the long-term work, since at that time we'll be creating more scopes, and so will document all of them at once.